### DISCUSSION TITLE
Issue på mobile

### DISCUSSION THREAD

**Discussion by arnestar** [rndfm/elspotpris#160] [General]
iPhone XS iOS 16.4, Display zoom: Default,  Safari Mobile, Chrome Mobile 
Jeg var inde på siden via min iPhone og bemærkede at "Filtre" boksen ikke kommer frem i portrait mode. 
Boksen kommer selvfølgelig frem ved at trække til venstre men tænker at mange overser boksen  og de to options, medmindre man ved den er.
Boksen er også det eneste der er placeret "udenfor" skærmen. 
Alt virker i landscape.

**Comment by rndfm** [comment]
Takker. Den havde jeg ikke lige spottet. Den er nu fikset.
